src/blog/_posts/2018-11-06-opera-57.md

```diff
@@ -0,0 +1,151 @@
+---
+title: What's new in Chromium 70 and Opera 57
+authors:
+- fredrik-soderqvist
+intro: 'Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable). Here’s what it means for web developers.'
+tags:
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable).
+Here’s what it means for web developers.
+
+## Displaying a dialog will cause pages to leave fullscreen
+
+Dialogs (authentication prompt, payment request et.c) require proper context
```

Comment:
irregular punctuation for etc
https://data.grammarbook.com/blog/definitions/all-about-etc/

Reply:
Fixed.